webapp/channels/src/components/move_thread_modal/move_thread_modal.tsx

```diff
@@ -0,0 +1,256 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+import React, {useCallback, useRef, useState} from 'react';
+
+import {FormattedMessage, useIntl} from 'react-intl';
+
+import {useSelector} from 'react-redux';
+
+import {ValueType} from 'react-select';
+
+import classNames from 'classnames';
+
+import {makeGetChannel} from 'mattermost-redux/selectors/entities/channels';
+import {getCurrentTeam} from 'mattermost-redux/selectors/entities/teams';
+
+import NotificationBox from 'components/notification_box';
+
+import {GlobalState} from 'types/store';
+
+import Constants from 'utils/constants';
+
+import PostMessagePreview from 'components/post_view/post_message_preview';
+import {GenericModal} from '@mattermost/components';
+
+import {ActionResult} from 'mattermost-redux/types/actions';
+
+import {Post, PostPreviewMetadata} from '@mattermost/types/posts';
+import {Channel} from '@mattermost/types/channels';
+
+import ForwardPostChannelSelect, {ChannelOption} from '../forward_post_modal/forward_post_channel_select';
```

Comment:
This seems wrong, in the sense that we are from a component using part of some other component. I would refactor this to its own component under the component folder.

In any case... the import should be as `components/...` instead of `../...`.

Reply:
I've decided to revert this for now. The selector component is receiving changes and it's causing merge conflicts that are a bit hard to keep on top of already. Can refactor this later if we think it's necessary